<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mohaema</class>
 <widget class="QMainWindow" name="Mohaema">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>622</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>182</red>
        <green>175</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>218</red>
        <green>215</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>91</red>
        <green>87</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>121</red>
        <green>117</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>235</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>182</red>
        <green>175</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>218</red>
        <green>215</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>182</red>
        <green>175</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>218</red>
        <green>215</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>91</red>
        <green>87</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>121</red>
        <green>117</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>235</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>182</red>
        <green>175</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>218</red>
        <green>215</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>91</red>
        <green>87</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>182</red>
        <green>175</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>218</red>
        <green>215</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>91</red>
        <green>87</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>121</red>
        <green>117</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>91</red>
        <green>87</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>91</red>
        <green>87</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>182</red>
        <green>175</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>182</red>
        <green>175</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>182</red>
        <green>175</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Mohaema segmenter</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="main">
   <widget class="QPushButton" name="addword">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>530</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>添加词库</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>480</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始分词</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Output">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>34</y>
      <width>361</width>
      <height>426</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="frameShape">
     <enum>QFrame::VLine</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>480</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>识别姓名</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>530</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>显示词性</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="Left">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Text</string>
     </attribute>
     <widget class="QTextEdit" name="Input">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>351</width>
        <height>431</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>235</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>235</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>182</red>
            <green>175</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="frameShape">
       <enum>QFrame::VLine</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>File</string>
     </attribute>
     <widget class="QPushButton" name="AddFile">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>170</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Add a File</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>351</width>
        <height>431</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::VLine</enum>
      </property>
     </widget>
     <zorder>textBrowser</zorder>
     <zorder>AddFile</zorder>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>480</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ruthless Drippin ONE</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Mohaema</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="time_speed">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>611</width>
      <height>31</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>182</red>
          <green>175</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>182</red>
          <green>175</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>182</red>
          <green>175</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'楷体'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;欢迎使用Mahaema Segmenter!耗时:0.0000s     分词速度:0.0000词/s&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>753</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="import_1"/>
    <addaction name="Go"/>
    <addaction name="SaveFile"/>
    <addaction name="Clear"/>
    <addaction name="separator"/>
    <addaction name="Exit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>编辑(E)</string>
    </property>
    <addaction name="action_Ctrl_Z"/>
    <addaction name="action_Ctrl_Y"/>
    <addaction name="action_Ctrl_X"/>
    <addaction name="action_Ctrl_C"/>
    <addaction name="action_Ctrl_V"/>
   </widget>
   <widget class="QMenu" name="menu_W">
    <property name="title">
     <string>显示 (D)</string>
    </property>
    <widget class="QMenu" name="segmentpattern">
     <property name="title">
      <string>分隔符</string>
     </property>
     <addaction name="space_"/>
     <addaction name="vertical_"/>
     <addaction name="left_sider"/>
    </widget>
    <addaction name="segmentpattern"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助 (H)</string>
    </property>
    <addaction name="get_help"/>
    <addaction name="about_"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu"/>
   <addaction name="menu_W"/>
   <addaction name="menu_H"/>
  </widget>
  <action name="action_7">
   <property name="text">
    <string>剪切 (ctrl+X)</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>自动换行</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>查看帮助</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>字体</string>
   </property>
  </action>
  <action name="import_1">
   <property name="icon">
    <iconset>
     <normaloff>New Icons/ICO/Add.ico</normaloff>New Icons/ICO/Add.ico</iconset>
   </property>
   <property name="text">
    <string>导入文本(Ctrl+I)</string>
   </property>
   <property name="whatsThis">
    <string>Import a file as input</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="SaveFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>New Icons/save.png</normaloff>New Icons/save.png</iconset>
   </property>
   <property name="text">
    <string>保存文本(Ctrl+S)</string>
   </property>
   <property name="whatsThis">
    <string>Save output text as a new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="Go">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>New Icons/png/48x48/play.png</normaloff>New Icons/png/48x48/play.png</iconset>
   </property>
   <property name="text">
    <string>开始分词(Ctrl+G)</string>
   </property>
   <property name="whatsThis">
    <string>Start segment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_Ctrl_Z">
   <property name="icon">
    <iconset>
     <normaloff>New Icons/03/16/13.png</normaloff>New Icons/03/16/13.png</iconset>
   </property>
   <property name="text">
    <string>撤销 (Ctrl+Z)</string>
   </property>
   <property name="whatsThis">
    <string>redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Ctrl_C">
   <property name="icon">
    <iconset>
     <normaloff>New Icons/ICO/Copy.ico</normaloff>New Icons/ICO/Copy.ico</iconset>
   </property>
   <property name="text">
    <string>复制 (Ctrl+C)</string>
   </property>
   <property name="whatsThis">
    <string>copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Ctrl_V">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>New Icons/ICO/Paste.ico</normaloff>New Icons/ICO/Paste.ico</iconset>
   </property>
   <property name="text">
    <string>粘贴 (Ctrl+V)</string>
   </property>
   <property name="whatsThis">
    <string>paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_del">
   <property name="icon">
    <iconset>
     <normaloff>New Icons/ICO/Delete_01.ico</normaloff>New Icons/ICO/Delete_01.ico</iconset>
   </property>
   <property name="text">
    <string>删除 (del)</string>
   </property>
   <property name="whatsThis">
    <string>delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_16">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>New Icons/02/16/03.png</normaloff>New Icons/02/16/03.png</iconset>
   </property>
   <property name="text">
    <string>显示运行时间</string>
   </property>
   <property name="whatsThis">
    <string>show running time</string>
   </property>
  </action>
  <action name="action_17">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>New Icons/ICO/Chart.ico</normaloff>New Icons/ICO/Chart.ico</iconset>
   </property>
   <property name="text">
    <string>显示分词速度</string>
   </property>
   <property name="whatsThis">
    <string>show segmenting speed</string>
   </property>
  </action>
  <action name="space_">
   <property name="text">
    <string>空格</string>
   </property>
  </action>
  <action name="star_">
   <property name="text">
    <string>*</string>
   </property>
  </action>
  <action name="left_sider">
   <property name="text">
    <string>/</string>
   </property>
  </action>
  <action name="right_sider">
   <property name="text">
    <string>\</string>
   </property>
  </action>
  <action name="Exit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>New Icons/ICO/Door.ico</normaloff>New Icons/ICO/Door.ico</iconset>
   </property>
   <property name="text">
    <string>退出 (Alt+F4)</string>
   </property>
   <property name="whatsThis">
    <string>Exit application</string>
   </property>
  </action>
  <action name="get_help">
   <property name="icon">
    <iconset>
     <normaloff>New Icons/help.png</normaloff>New Icons/help.png</iconset>
   </property>
   <property name="text">
    <string>查看帮助</string>
   </property>
   <property name="whatsThis">
    <string>help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="about_">
   <property name="icon">
    <iconset>
     <normaloff>New Icons/ICO/User Info.ico</normaloff>New Icons/ICO/User Info.ico</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="Clear">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>New Icons/png/48x48/note_remove.png</normaloff>New Icons/png/48x48/note_remove.png</iconset>
   </property>
   <property name="text">
    <string>清空文本 (Ctrl+R)</string>
   </property>
   <property name="whatsThis">
    <string>clear text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_Ctrl_X">
   <property name="icon">
    <iconset>
     <normaloff>New Icons/13341408.png</normaloff>New Icons/13341408.png</iconset>
   </property>
   <property name="text">
    <string>剪切 (Ctrl+X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Ctrl_Y">
   <property name="icon">
    <iconset>
     <normaloff>New Icons/03/16/15.png</normaloff>New Icons/03/16/15.png</iconset>
   </property>
   <property name="text">
    <string>恢复 (Ctrl+Y)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="vertical_">
   <property name="text">
    <string>|</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
